libraries/default.rb

```diff
@@ -39,6 +39,15 @@ def get_seed_file
       node.fetch(:cluster_seed_file, 'cluster_seed.tgz')
     end
 
+    def on_aws?
+      if node['vagrant_environment'] == true
+        Chef::Log.info "deploying to a vagrant cluster"
```

Comment:
@lbjay  Yeah, this throws out a log entry when deploying to vagrant clusters. Probably not needed any more.